schema.ddl

```diff
@@ -51,18 +118,22 @@ ON space (provider_id);
 CREATE UNIQUE INDEX IF NOT EXISTS idx_space_id
 ON space (id);
 
+/*
+ * Subnet
+ */
 CREATE TABLE IF NOT EXISTS subnet (
 	uuid				TEXT PRIMARY KEY,
 	id					TEXT NOT NULL UNIQUE,
 	life				INT,
 	provider_id			TEXT,
-	provider_network_id	TEXT,
 	cidr				TEXT,
 	vlantag				INT,
 	is_public			BOOLEAN,
 	space_id			TEXT NOT NULL,
 	fan_local_underlay	TEXT,
 	fan_overlay			TEXT,
+	-- provider_network_id
```

Comment:
For this, I think we can introduce an new table, network with uuid, id (as in provider ID) and name. Then this can have a FK constraint.